src/explore-education-statistics-admin/src/pages/release/pre-release/__tests__/PreReleasePageContainer.test.tsx

```diff
@@ -76,9 +76,10 @@ describe('PreReleasePageContainer', () => {
       }),
     ).toBeInTheDocument();
 
+    // Need to match both 09:00 and 10:00, because mocking out the "London/Europe" timezone for BST/GMT testing isn't really feasible
```

Comment:
We mock date-fns here https://github.com/dfe-analytical-services/explore-education-statistics/blob/dev/src/explore-education-statistics-common/test/mockDate.ts, could it be extended or something similar done to mock the timezone stuff?

Reply:
That's a good shout, I'll have a go. What I'm tying to avoid here is this test suddenly failing in 6 months time when we enter GMT, so whatever mock we write would need to know when the timezones change. Not sure if that's the same date each year, so will go research